docs/troubleshooting.md

```diff
@@ -0,0 +1,61 @@
+# Troubleshooting Posit Publisher
+
+This document contains some common issues and solutions for Posit Publisher.
+
+### The Configuration has a schema error
```

Comment:
Minor nit, but do we want to go from level 1 to level 3 headings like this? Or should this (and the ones below) be level 2?

Reply:
Good callout fixed in [e9be268](https://github.com/posit-dev/publisher/pull/2546/commits/e9be2685cf2d3e1e20824a63170db274b2fe65d9)